### PR TITLE
Fix tracking helpers TS errors

### PR DESCRIPTION
**Pull request by marialani**
## What are you doing in this PR?
This fixes Typescript and linting errors in the `helpers/tracking` subfolder.

[**Trello Card**](https://trello.com/c/USbbS4E3/209-typescript-error-busting-helpers-tracking)

### CODE REVIEW

File: support-frontend/assets/helpers/tracking/acquisitions.ts [hunk 1]
```diff
@@ -36,33 +36,33 @@ export type OphanIds = {
...
-	referrerUrl: string | null | undefined;
-	campaignCodes: string[] | null | undefined;
-	componentId: string | null | undefined;
-	componentType: string | null | undefined;
-	source: string | null | undefined;
-	abTests: AcquisitionABTest[] | null | undefined;
-	gaId: string | null | undefined;
-	queryParameters: AcquisitionQueryParameters | null | undefined;
-	labels: string[] | null | undefined;
+	visitId?: string | null;
```
Comment (rupertbates) [inline support-frontend/assets/helpers/tracking/acquisitions.ts:53]: Do we still need all the nulls here? Could we restrict it to `?: string`?

Comment (marialani) [reply support-frontend/assets/helpers/tracking/acquisitions.ts:53]: I'm not too sure, I'd need to go on a bit of a hunt in the codebase for anywhere that we may explicitly be setting one of these values to null

**Review by rupertbates — APPROVED** [review]
👍  Just one question about types

File: support-frontend/assets/helpers/tracking/acquisitions.ts (hunk 1 above)
Comment (marialani) [reply support-frontend/assets/helpers/tracking/acquisitions.ts:53]: We could get rid a fair few of the nulls here 👍, we only need a few for props that call a shared helper function

**Pull request merged**